tests/rt.conf

```diff
@@ -242,6 +242,21 @@ RUN     | datm_cdeps_gfs
 COMPILE | -DAPP=NG-GODAS -DDEBUG=ON                                                                                               | - wcoss2.intel acorn.intel              | fv3 |
 RUN     | datm_cdeps_debug_cfsr                                                                                                   | - wcoss2.intel acorn.intel              | fv3 |
 
+###################################################################################################################################################################################
+# CDEPS Data Atmosphere tests with LND                                                                                                                                            #
+###################################################################################################################################################################################
+
+COMPILE | -DAPP=LND                                                                                                               | - wcoss_cray                            | fv3 |
+RUN     | datm_cdeps_lnd_gswp3                                                                                                    | - wcoss_cray                            | fv3 |
+RUN     | datm_cdeps_lnd_gswp3_rst                                                                                                | - wcoss_cray                            |     | datm_cdeps_lnd_gswp3
```

Comment:
Is this old code or a typo. wcoss_cray no longer exists, did you mean wcoss2.intel?

Reply:
Most of the datm configuration is not available for wcoss2.intel and I could also make some for these data atmosphere configurations. wcoss_cray is definitely old code and need to be changed to wcoss2.intel. Let me quickly fix it.

Reply:
done.